.github/workflows/test-action.yml

```
@@ -0,0 +1,61 @@
+---
+name: Test ansible-test-gh-action
+on:
+  push:
+    branches: [main]
+  pull_request:
+
+jobs:
+  tests:
+    name: Basic tests
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+        - ansible-core-version: stable-2.12
+          docker-image: default
+          git-checkout-ref:
+          pre-test-cmd:
+          python-version: '3.8'
+          target:
+          target-python-version:
+          testing-type: sanity
+          test-deps:
+        - ansible-core-version: devel
+          docker-image: default
+          git-checkout-ref:
+          pre-test-cmd:
+          python-version: '3.9'
+          target:
+          target-python-version:
+          testing-type: units
+          test-deps:
+        - ansible-core-version: stable-2.13
+          docker-image: default
+          git-checkout-ref:
+          pre-test-cmd:
+          python-version: '3.10'
+          target:
+          target-python-version:
+          testing-type: integration
+          test-deps:
```

Just realized there's too much duplication here. Let's fix this:
```suggestion
        docker-image:
        - default
        git-checkout-ref:
        -
        pre-test-cmd:
        -
        target:
        -
        target-python-version:
        -
        test-deps:
        -
        include:
        - ansible-core-version: stable-2.12
          python-version: '3.8'
          testing-type: sanity
        - ansible-core-version: devel
          python-version: '3.9'
          testing-type: units
        - ansible-core-version: stable-2.13
          python-version: '3.10'
          testing-type: integration
```

---

This change caused the matrix to collapse to one entry (https://github.com/felixfontein/ansible-test-gh-action/runs/7635270423?check_suite_focus=true).